scripts/run_example

```diff
@@ -115,29 +115,16 @@ do
   case "${client_variant}" in
     cargo)
       rust_client_manifest="${SCRIPTS_DIR}/../examples/${EXAMPLE}/client/rust/Cargo.toml"
-      if [[ -f "${rust_client_manifest}" ]]; then
           cargo run --release --manifest-path="${rust_client_manifest}" -- \
             --root-tls-certificate="${SCRIPTS_DIR}/../examples/certs/local/ca.pem" \
             "${CLIENT_ARGS[@]-}"
-      else
-        echo "The ${EXAMPLE} example does not contain a ${client_variant} client. Skipping this client."
-      fi
       ;;
     bazel)
-      build_file="${SCRIPTS_DIR}/../examples/${EXAMPLE}/client/BUILD"
-      if [[ -f "${build_file}" ]]; then
           "${SCRIPTS_DIR}/../bazel-client-bin/examples/${EXAMPLE}/client/client" "${TLS_ARGS[@]}" "${ADDITIONAL_ARGS[@]-}" "${CLIENT_ARGS[@]-}"
-      else
-        echo "The ${EXAMPLE} example does not contain a ${client_variant} client. Skipping this client."
-      fi
       ;;
     npm)
       npm_client="${SCRIPTS_DIR}/../examples/${EXAMPLE}/client/nodejs"
-      if [[ -d "${npm_client}" ]]; then
           npm start --prefix "${npm_client}"
-      else
-        echo "The ${EXAMPLE} example does not contain a ${client_variant} client. Skipping this client."
-      fi
       ;;
     *)
       echo "Invalid example client variant: ${client_variant}"
```

Comment:
Which example variant will be run, if nothing is specified in the `-c`?
Will it just be Bazel?
https://github.com/project-oak/oak/blob/e9db4c6f4b342bc6066fccb71104210b4984dbb3/scripts/run_example#L9

Reply:
Yup, since it's the default. :) https://github.com/project-oak/oak/blob/018389a2f8c1080be13eea0ec248747dd366505f/scripts/run_example#L32

We can change that if we'd like to run all available example by default, but that seems like a separate task :)